proto/inx.proto

```diff
@@ -60,12 +67,15 @@ message BaseToken {
 }
 
 message NodeStatus {
+  reserved 2, 3;
```

Comment:
Do you know why we reserved those two?
Why not do a breaking change and "count correctly" ?

Reply:
Those were not used before, and since those cannot be re-used I added it to make it explicit. I don't know why those are reserved though. We could make a breaking change, but not as part of this PR.